src/AccessibilityInsights.SharedUx/Enums/HighlighterColor.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+namespace AccessibilityInsights.SharedUx.Enums
+{
+    /// <summary>
+    /// These values (as strings) correspond to keys of SolidColorBrushes 
+    /// found in the HighlightStyle.xaml resource dictionary
```

Comment:
There is also no use of `SnapshotRootBrush` anymore, so you could take that out if you want.